modules/cluster-logging-collector-log-forwarding-about.adoc

```diff
@@ -128,8 +128,8 @@ spec:
 <6> Configuration for an input to filter application logs from the `my-namespace` project.
 <7> Configuration for a pipeline to send audit logs to the secure external Elasticsearch instance:
 ** Optional. A name to describe the pipeline.
-** The `inputRefs` is the log type, in this example `audit`.
-** The `outputRefs` is the name of the output to use, in this example `elasticsearch-secure` to forward to the secure Elasticsearch instance and `default` to forward to the internal Elasticsearch instance.
+** The `inputRefs` are the log types that are inputs for this pipeline, such as `audit` in this example.
```

Comment:
The `inputRefs` are --> The `inputRefs` parameters are

Suggest to always follow the guidelines here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-object-references

Or, you can write like this: Log types that are inputs for this pipeline, such as `audit` in this example.